pkg/adsc/adsc.go

```diff
@@ -489,16 +489,19 @@ func (a *ADSC) handleRecv() {
 		if err != nil {
 			a.RecvWg.Done()
 			adscLog.Infof("Connection closed for node %v with err: %v", a.nodeID, err)
-			a.errChan <- err
 			// if 'reconnect' enabled - schedule a new Run
-			if a.cfg.BackoffPolicy != nil {
-				time.AfterFunc(a.cfg.BackoffPolicy.NextBackOff(), a.reconnect)
-			} else {
-				a.Close()
-				a.WaitClear()
-				a.Updates <- ""
-				a.XDSUpdates <- nil
-				close(a.errChan)
+			select {
+			case a.errChan <- err:
```

Comment:
this means if we send an err we will NOT reconnect? that doesn't seem right

Reply:
I mean the receiver who handles the errChan does the reconnection action, but it seems like not very resasonable.